docker/Dockerfile-build-backend.jvm

```diff
@@ -1,42 +1,32 @@
-####
-# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
-###
-
-# Build the project
 FROM registry.access.redhat.com/ubi8/openjdk-11:latest AS build
 USER root
 COPY . /home/jboss
 WORKDIR /home/jboss
 RUN ./mvnw clean package -DskipTests --projects :checkstyle,:notifications-backend
 
-# Build the container
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 
 ARG JAVA_PACKAGE=java-11-openjdk-headless
 ARG RUN_JAVA_VERSION=1.3.8
```

Comment:
No longer needed

Reply:
the `ARG RUN_JAVA_VERSION=1.3.8` I meant. The comment could/should still be there as in aggregator.

Reply:
> the `ARG RUN_JAVA_VERSION=1.3.8` I meant. The comment could/should still be there as in aggregator.

what's wrong with ARG RUN_JAVA_VERSION?

Reply:
A few lines down you remove its usage.

Reply:
removed the argument